pkgs/by-name/py/pyprland/package.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, fetchFromGitHub
+, python311
+}:
```

When the list is so short i think it better to write in one line.
```suggestion
{ lib, fetchFromGitHub, python311 }:
```